crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/controller/CrownCourtProceedingController.java

```diff
@@ -56,15 +55,16 @@ private CrownCourtActionsRequestDTO preProcessRequest(ApiCheckCrownCourtActionsR
                     schema = @Schema(implementation = ErrorDTO.class)
             )
     )
-    public ResponseEntity<ApiCheckCrownCourtActionsResponse> checkCrownCourtActions(@Parameter(description = "Check Crown Court Actions data",
-            content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
-                    schema = @Schema(implementation = ApiCheckCrownCourtActionsRequest.class)
-            )
-    ) @Valid @RequestBody ApiCheckCrownCourtActionsRequest crownCourtActionsRequest) {
+    public ResponseEntity<ApiProcessRepOrderResponse> processCrownRepOrder(
```

Comment:
Maybe "processCrownCourtRepOrder" more appropriate method name?

Reply:
Honestly, I was debating simply changing it to: `processRepOrder`, my thinking is that since we're in the Crown Court Proceeding service the context should be fairly obvious. How would you feel about that change?

Reply:
Yeah, either processRepOrder or processCrownCourtRepOrder would be nice and we can update the description accordingly. As you have put so much effort in renaming I thought we should make it perfect :-)

Reply:
Changed to `processRepOrder`